pkgs/by-name/pi/pihole/package.nix

```
@@ -0,0 +1,259 @@
+{
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
```

```suggestion
  makeBinaryWrapper,
```

Let's use the preferred way, as recommended in the nix manual.